keyboards/preonic/keymaps/isouk/keymap.c

```diff
@@ -0,0 +1,385 @@
+ * | Esc  |   A  |   O  |   E  |   U  |   I  |   D  |   H  |   T  |   N  |   S  |  /   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   ;  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |   Z  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Brite| Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_DVORAK] = LAYOUT_preonic_grid( \
+  KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC, \
+  KC_TAB,  KC_QUOT, KC_COMM, KC_DOT,  KC_P,    KC_Y,    KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    KC_DEL,  \
+  KC_ESC,  KC_A,    KC_O,    KC_E,    KC_U,    KC_I,    KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_SLSH, \
+  KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,    KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_ENT,  \
+  BACKLIT, KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT  \
+),
+
+    /* Lower
+     * ,-----------------------------------------------------------------------------------.
+     * |   ~  |   !  |   @  |   #  |laltf4|   %  |   ^  |   &  |   *  |   {  |   }  |      |
+     * |------+------+------+------+------+-------------+------+------+------+------+------|
+     * |   ~  |  F1  |  F2  |  F3  |  F4  |   %  |   ^  |   &  |   *  |   (  |   )  |      |
+     * |------+------+------+------+------+-------------+------+------+------+------+------|
+     * | Caps |  F5  |  F6  |  F7  |  F8  |      |      |   _  |   =  |   {  |   }  |  |   |
+     * |------+------+------+------+------+------|------+------+------+------+------+------|
+     * |   \  |  F9  |  F10 |  F11 |  F12 |      | iso #|iso \ |End | | Home |      |   ~  |
+     * |------+------+------+------+------+------+------+------+------+------+------+------|
+     * |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+     * `-----------------------------------------------------------------------------------'
+     */
+[_LOWER] = LAYOUT_preonic_grid( \
+        RELOAD,  BEST,    SECTION, KC_HASH, LALT(KC_F4),  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, LSFT(KC_LBRC), LSFT(KC_RBRC), KC_PSCR, \
+        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,         KC_NO,         KC_NO, \
+        KC_CAPS, KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_NO,   KC_NO,   KC_UNDS, KC_EQL,  KC_NO,         KC_NO,         KC_NO, \
+        KC_NUBS, KC_F9,   KC_F10,  KC_F11,  KC_F12,       KC_NO,   KC_NUHS, KC_NUBS, KC_HOME, KC_END,        KC_NO,         LSFT(KC_NUHS), \
+        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_TRNS,      KC_NO,   KC_NO,   KC_TRNS, KC_MNXT, KC_VOLD,       KC_VOLU,       KC_MPLY \
```

Comment:
```suggestiob
[_LOWER] = LAYOUT_preonic_grid(
        RELOAD,  BEST,    SECTION, KC_HASH, LALT(KC_F4),  KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, LSFT(KC_LBRC), LSFT(KC_RBRC), KC_PSCR,
        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_NO,         KC_NO,         KC_NO,
        KC_CAPS, KC_F5,   KC_F6,   KC_F7,   KC_F8,        KC_NO,   KC_NO,   KC_UNDS, KC_EQL,  KC_NO,         KC_NO,         KC_NO,
        KC_NUBS, KC_F9,   KC_F10,  KC_F11,  KC_F12,       KC_NO,   KC_NUHS, KC_NUBS, KC_HOME, KC_END,        KC_NO,         LSFT(KC_NUHS),
        KC_NO,   KC_NO,   KC_NO,   KC_NO,   KC_TRNS,      KC_NO,   KC_NO,   KC_TRNS, KC_MNXT, KC_VOLD,       KC_VOLU,       KC_MPLY
```

Reply:
Is there a mistake here? i don't see any changes.

Reply:
Fixed.